packer/includes/Vagrantfile

```diff
@@ -0,0 +1,169 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+require 'yaml'
+
+Vagrant.require_version '>= 1.8.0'
+
+# Find the Vagrant ID if supplied.
+id = nil
+ARGV.each do |arg|
+  id = arg[/\h{7}/] ? arg : id
+end
+
+# Determine the current working directory.
+cwd = nil
+if id != nil
+  cwd = /#{id}.*?(\/.*?)\n/.match(`vagrant global-status`)[1]
```

Comment:
@Decipher only issue is when referencing another VM via it's `[ID]` the `cwd` gets trailing whitespace and then can't find the project `Vagrantfile`.

Simple fix is to add `.strip` to this line.